auctions/templates/bid_list.html

```diff
@@ -6,28 +6,38 @@
         <h2>Bids</h2>
     </div>
     <div class="row">
-    <table class="hover">
+    <table>
         <thead>
             <tr>
+                <th width="100">Created</th>
                 <th>Issue</th>
-                <th>Ask</th>
-                <th>Offer</th>
-                <th>Fees</th>
-                <th>Total Charge</th>
+                <th width="100">Ask</th>
+                <th width="200">Offer</th>
             </tr>
         </thead>
         <tbody>
     {% for bid in bids %}
-            {% for offer in bid.offers %}
             <tr>
+                <td>
+                    <span data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="{{ bid.created|date:"c" }}">{{ bid.created|date:"M j" }}</span></td>
                 <td><a href="{{bid.url}}">{{bid.issue.title}}</a></td>
-                <td>{{ bid.ask }}</td>
-                <td>{{ bid.offer }}</td>
-                <td>{% for fee in offer.fees %}
-                    <strong>{{ fee.fee_type }}</strong>: {{ fee.amount }}<br/>
-                    {% endfor %}
+                <td>
+                    <span
+                    {% if bid.ask_met %}
+                        data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="Your ask has been met"
+                    {% endif %}
+                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}
```

Comment:
Not sure what you mean ... it's on the list view in the column before the offer column.
![codesy-bid-ask](https://cloud.githubusercontent.com/assets/71928/15291438/826913a6-1b43-11e6-95b1-be72a7125408.png)